src/analysis/calculus/fderiv/exp.lean

```diff
@@ -0,0 +1,106 @@
+/-
+Copyright (c) 2023 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import measure_theory.integral.fund_thm_calculus
+import analysis.normed_space.exponential
+import analysis.quaternion
+import algebra.lie.of_associative
+import analysis.special_functions.exponential
+import analysis.calculus.fderiv_symmetric
+
+/-! More lemmas aboutderiviatives of `exp`.
+
+This follows https://physics.stackexchange.com/a/41671/185147. -/
+
+variables {𝔸 𝔹 : Type*}
+variables [normed_ring 𝔸] [normed_algebra ℝ 𝔸] [complete_space 𝔸]
+
+-- to make the goal view readable
+notation (name := deriv) `∂` binders `, ` r:(scoped:67 f, deriv f) := r
+local notation `e` := exp ℝ
+
+lemma has_deriv_at_exp_smul_const (A : 𝔸) (t : ℝ) :
+  has_deriv_at (λ t : ℝ, exp ℝ (t • A)) (A * exp ℝ (t • A)) t := sorry
```

Comment:
@ADedecker; do you know if your existing results in `nalysis.special_functions.exponential` are strong enough to prove this?

Reply:
I was able to revive your old proof with minimal modification. The primed version now follows trivially from the unprimed one.

Is there any advantage to switching to your proposed strategy?

Reply:
>  (1) prove that exp behaves well under continuous algebra morphisms

I proved this a while ago, it's `map_exp`.

Reply:
I've extracted all these results to #19062